widgets/contact/contact.php

```diff
@@ -343,6 +343,72 @@ function get_widget_form(){
 						),
 					),
 
+					'fields' => array(
+						'type'   => 'section',
+						'label'  => __( 'Fields', 'so-widgets-bundle' ),
+						'fields' => array(
+							'font' => array(
+								'type'    => 'font',
```

Comment:
Yeah, I'll maybe take a crack at simplifying this soon. I think there might still be an issue about this somewhere. :sweat_smile: